README.md

```diff
@@ -121,6 +122,7 @@ This plugin provides the following custom rules, which are included as appropria
 - [restrict-full-import](docs/rules/restrict-full-import.md): Prevents importing the entirety of a package.
 - [sinon-no-restricted-features](docs/rules/sinon-no-restricted-features.md): Restricts the use of specified sinon features.
 - [sinon-prefer-meaningful-assertions](docs/rules/sinon-prefer-meaningful-assertions.md): Requires the use of meaningful sinon assertions through sinon.assert or sinon-chai.
+- [no-unnamed-dynamic-imports](docs/rules/no-unnamed-dynamic-imports.md): Requires that all dynamic imports contain a `webpackChunkName` comment.
```

Comment:
What do you think about prefixing these with webpack-/sewing-kit-?

Reply:
Done.